packages/bundler-plugin-core/src/utils/normalizeOptions.ts

```diff
@@ -56,6 +56,32 @@ const UploadOverridesSchema = z.object({
     .optional(),
 });
 
+const OIDCSchema = (options: Options) =>
+  z.object(
+    {
+      useGitHubOIDC: z
+        .boolean({
+          invalid_type_error: "`useGitHubOIDC` must be a boolean.",
+        })
+        .default(false),
+      /**
+       * Following along with how we handle this in our GH Action.
+       *
+       * See: https://github.com/codecov/codecov-action/blob/main/src/buildExec.ts#L53-L58
+       */
+      OIDCEndpoint: z
+        .string({
+          invalid_type_error: "`OIDCEndpoint` must be a string.",
+        })
+        .url({
```

Comment:
Technically a JWT [audience](https://www.rfc-editor.org/rfc/rfc7519.html#section-4.1.3) is StringOrURI so I think we can loosen this requirement that it must be a url